packages/editor/src/components/rich-text/tinymce.js

```diff
@@ -167,7 +168,16 @@ export default class TinyMCE extends Component {
 		}
 	}
 
+	/**
+	 * Initializes TinyMCE. Can only be called once per instance.
+	 */
 	initialize() {
+		if ( this.initialize.called ) {
```

Comment:
Why not just `this.initialized` 🤷‍♂️ 

I find it a bit weird to attach a property to the method while we have access to the object but it's minor.

Reply:
It's local state. But I don't mind either way.